prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -134,6 +134,25 @@ public async Task<IEnumerable<EnrolleeListViewModel>> GetEnrolleesAsync(Enrollee
                 .ToListAsync();
         }
 
+        public async Task<EnrolleeNavigation> GetAdjacentEnrolleeIdAsync(int enrolleeId)
+        {
+            var nextId = await _context.Enrollees
+                .Where(e => e.Id > enrolleeId)
+                .OrderBy(e => e.Id)
+                .Select(e => e.Id)
+                .FirstOrDefaultAsync();
+
+            var previousId = await _context.Enrollees
+                .Where(e => e.Id < enrolleeId)
+                .OrderByDescending(e => e.Id)
+                .Select(e => e.Id)
+                .FirstOrDefaultAsync();
+
+            return await _context.Enrollees
```

Comment:
you dont have to call into the context again here, you already have the two IDs you need